web/src/AssetEditPage.js

```diff
@@ -262,6 +269,44 @@ class AssetEditPage extends React.Component {
             </Col>
           </Row>
         }
+        <Row style={{marginTop: "20px"}} >
+          <Col style={{marginTop: "5px"}} span={(Setting.isMobile()) ? 22 : 2}>
+            {Setting.getLabel(i18next.t("general:Intranet"), i18next.t("general:Intranet - Tooltip"))} :
+          </Col>
+          <Switch checked={isIntranet} onChange={checked => {
+            this.setState({
+              isIntranet: checked,
+            });
+          }}>{i18next.t("general:Intranet")}
+          </Switch>
+        </Row>
+        {
+          isIntranet && (
+            <React.Fragment>
+              <Row style={{marginTop: "20px"}} >
+                <Col style={{marginTop: "5px"}} span={(Setting.isMobile()) ? 22 : 2}>
+                  {Setting.getLabel(i18next.t("Asset:Hostname"), i18next.t("Asset:Hostname - Tooltip"))} :
+                </Col>
+                <Col span={22} >
+                  <Input value={asset.hostname} onChange={e => {
+                    this.updateAssetField("hostname", e.target.value);
+                  }} />
+                </Col>
+              </Row>
+              <Row style={{marginTop: "20px"}} >
+                <Col style={{marginTop: "5px"}} span={(Setting.isMobile()) ? 22 : 2}>
+                  {Setting.getLabel(i18next.t("Asset:Remote hostname"), i18next.t("Asset:Remote hostname - Tooltip"))} :
+                </Col>
+                <Col span={22} >
+                  <Input value={asset.remoteHostname || Setting.ServerUrl} onChange={e => {
```

Comment:
If I change the `remoteHostname`'s value now, will it work immediately? Or need to restart Go backend?